code/game/turfs/open/space/space.dm

```diff
@@ -58,7 +58,7 @@ GLOBAL_REAL_VAR(space_appearances) = make_space_appearances()
 
 	initialized = TRUE
 
-	if(!loc:area_has_base_lighting) //Only provide your own lighting if the area doesn't for you
+	if(!loc.luminosity) //Only provide your own lighting if the area doesn't for you
```

Comment:
not a safe assumption areas default to lumin on, and non lit areas can have lumin, see area/New (unless we've drifted)

Reply:
When would it ever be valid for an area without base lighting to have luminosity? Seems to me like they should be intrinsically paired during `Initialize()`